lib/lightning/jobs/trigger.ex

```diff
@@ -35,9 +35,8 @@ defmodule Lightning.Jobs.Trigger do
     belongs_to :job, Job
     belongs_to :upstream_job, Job
 
-    field :type, Ecto.Enum,
-      values: @trigger_types,
-      default: :webhook
+    field :type, Ecto.Enum, values: @trigger_types
+    # default: :webhook
```

Comment:
@taylordowns2000 this is what I was mentioning [here](https://github.com/OpenFn/Lightning/pull/314#discussion_r961679662). Let's get @stuartc involved. I did show him that during one of our pairing sessions last week but I am not sure we did have a chance to discuss about.

The main reason why I decided to remove the default value is having would drop automatic changeset changes to be set when we create a trigger with type `webhook` (`webhook` being the default value there). That would be a problem to tests some scenarios like going from a `webhook` trigger to a `cron` one. I know that Ecto has some methods for forcing changesets changes and things like that but I didn't wanna take a solution that would add extra implementation time. At least now we can talk about this and decide what to do.

I am in favor of dropping the default because why should `webhook` be the default ? What is a default trigger type ? Are we trying to say that among all types `webhook` is the most natural one (for the user) ? These are my business reasoning when I am working on this.

What do you think guys ?